internal/pkg/bootstrap/bootstrap.go

```diff
@@ -4,6 +4,8 @@
 package bootstrap
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"encoding/json"
 	"errors"
```

Comment:
you could replace this package by `mainflux/errors`. Also, I'm not sure that you are using the error created at line 160